File: 02-java-security-features/skel/private/private.cpp
#include <iostream>

class Student {
public:
	int id;
	std::string name;
	void set_average(float a)
	{
		average = a;
	}
	float get_average()
	{
		return average;
	}
private:
	float average;
};

int main()
{
	Student s;
	s.id = 100;
	s.name = "Ionescu Andrei";

	s.average = 7.89;
	s.set_average(7.89);

	std::cout << "Average is: " << s.average << std::endl;
	std::cout << "Average is: " << s.get_average() << std::endl;

	return 0;
}

File: 02-java-security-features/skel/private/Private.java
class Student {
    public int id;
    public String name;
    private float average;

    public void setAverage(float average) {
        this.average = average;
    }

    public float getAverage() {
        return average;
    }
}

public class Private {
    public static void main(String[] args) {
        Student s = new Student();
        s.id = 100;
        s.name = "Ionescu Andrei";

        s.average = 7.89;
        s.setAverage(7.89f);
        System.out.println("average is " + s.average);
        System.out.println("average is " + s.getAverage());
    }
}
